strawberryfields/apps/graph/utils.py

```diff
@@ -153,3 +153,76 @@ def is_subgraph(subgraph: Iterable, graph: nx.Graph):
         return set(subgraph).issubset(graph.nodes)
     except TypeError:
         raise TypeError("subgraph and graph.nodes must be iterable")
+
+
+def is_clique(graph: nx.Graph) -> bool:
+    """Determines if the input graph is a clique
+
+    Args:
+        graph (nx.Graph): the input graph
+
+    Returns:
+        bool: returns ``True`` if input graph is a clique and ``False`` otherwise
+    """
+    edges = graph.edges
+    nodes = graph.order()
+
+    return len(edges) == nodes * (nodes - 1) / 2  # a clique has of n nodes has n*(n-1)/2 edges
+
+
+def c_0(subgraph: list, graph: nx.Graph):
+    """Generates the set C0 of nodes that are connected to all nodes in the input subgraph
```

Comment:
```suggestion
    """Generates the set :math:`C0` of nodes that are connected to all nodes in the input subgraph.
```

Reply:
```suggestion
    """Generates the set :math:`C_0` of nodes that are connected to all nodes in the input subgraph
```
Just checked the paper, they do capital C and 0 as a subscript, so I think :math:`C_0` makes sense in the docstrings. For the function name, how about `c_0` (so no caps)?

Reply:
I agree with :math:`C_0` for docstrings. For function usability, my feeling is that `c0` is preferable to `c_0`, just in terms of being less complex. Also an element of user preference there, if you count @jswinarton and I as users :laughing: